packages/hub/src/app/about/page.tsx

```diff
@@ -0,0 +1,58 @@
+import ReactMarkdown from "react-markdown";
+import { NarrowPageLayout } from "@/components/layout/NarrowPageLayout";
+import remarkGfm from "remark-gfm";
+import {
+  GITHUB_URL,
+  GITHUB_DISCUSSION_URL,
+  DISCORD_URL,
+  GRAPHQL_URL,
+  NEWSLETTER_URL,
+  QURI_DONATE_URL,
+} from "@/lib/common";
+
+const markdown = `
+# About Squiggle Hub
+Squiggle Hub serves as a communal platform for the creation and sharing of code written in [Squiggle](https://www.squiggle-language.com/). Both the platform and the source code are open-source and free to use.
+
+## Squiggle: A Glimpse
+[Squiggle](https://www.squiggle-language.com/) is a probabilistic estimation-oriented programming language that operates on the JavaScript platform. We think it's well-suited for intuition-based modeling and risk estimation. Additional details can be found on the Squiggle official website.
```

Comment:
I'd move the link from the first word to the last three words.

Reply:
Good point, done!